examples/sample_01/main.tf

```diff
@@ -0,0 +1,92 @@
+provider "google" {
+  credentials = file(var.credentials_file)
+  project = var.project
+  region      = "europe-west1"
+  zone        = "europe-west1-c"
+}
+
+resource "google_bigquery_dataset" "dataset" {
+  dataset_id                  = "example_dataset"
+  friendly_name               = "test"
+  description                 = "This is a test description"
```

Comment:
this is an optional parameter so either fill it with meaningful value or remove it. 
This is just a temporary dataset, so this is not really needed